tests/journald_tests.cpp

```diff
@@ -158,6 +158,7 @@ class JournaldLoggerTest : public MesosTest,
     modules = _modules.get();
 
     // Initialize the modules.
+    // TODO(akornatskyy): The following line crashes in Windows
```

Comment:
Is this still true? 
In my clean build, the only problem seems so be the mishandled substitution of `CMAKE_BUILD_TYPE` (see above); if I type that in manually into the `modules.json`, this line seems to work.

Reply:
no, removed